js/msp.js

```diff
@@ -406,11 +408,7 @@ var MSP = {
                     RC_tuning.RC_YAW_EXPO = parseFloat((data.getUint8(offset++) / 100).toFixed(2));
                     if (semver.gte(CONFIG.apiVersion, "1.20.0")) {
                         RC_tuning.rcYawRate = parseFloat((data.getUint8(offset++) / 100).toFixed(2));
-                    } else {
-                        RC_tuning.rcYawRate = 0;
```

Comment:
yea, it's just unnecessary now, since it's overwritten by: https://github.com/betaflight/betaflight-configurator/pull/161/files#diff-2ffd9d12c9aa7b46baff313cd6194e25R26